resources/monitoring/charts/grafana/values.yaml

```diff
@@ -63,6 +67,13 @@ image:
   # pullSecrets:
   #   - myRegistrKeySecretName
 
+testFramework:
```

Comment:
You can delete this part as we are not using the Grafana test framework.

Reply:
Done